distributions/stacks/ibm/application/katib/kustomization.yaml

```diff
@@ -2,33 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: kubeflow
 resources:
-  - ../../../../katib/installs/katib-standalone-ibm
-configMapGenerator:
-- name: kubeflow-config
-  envs:
-  - ../../config/params.env
-vars:
-# We need to define vars at the top level otherwise we will get
-# conflicts. 
-- fieldref:
-    fieldpath: metadata.namespace
-  name: katib-ui-namespace
-  objref:
-    kind: Service
-    name: katib-ui
-    apiVersion: v1
-- fieldref:
-    fieldPath: data.clusterDomain
-  name: clusterDomain
-  objref:
-    apiVersion: v1
-    kind: ConfigMap
-    name: kubeflow-config
-- fieldref:
-    fieldPath: metadata.namespace
-  name: namespace
-  objref:
-    apiVersion: v1
-    kind: ConfigMap
-    name: kubeflow-config
-    
\ No newline at end of file
+- ../../../../../apps/katib/upstream/installs/katib-with-kubeflow
```

Comment:
I am wondering, if we are not using [`katib-ibm`](https://github.com/kubeflow/katib/blob/master/manifests/v1beta1/installs/katib-ibm/kustomization.yaml) install for the IBM release, do we need to maintain it or we can just remove it from the upstream ?

Reply:
@Tomcli can answer that better. 
And thanks for catching the new line. Will update that immediately.

Reply:
@andreyvelich our old storageclass cannot run on mysql 8.0, so we have to patch it in the past. You can remove it from upstream now since we don't require it anymore.

Reply:
Thank you for this information!